backend/dataall/api/Objects/DatasetTable/resolvers.py

```diff
@@ -141,7 +141,9 @@ def preview(context, source, tableUri: str = None):
         )
         cursor = connection.cursor()
 
-        SQL = f'select * from "{table.GlueDatabaseName}"."{table.GlueTableName}" limit 50'  # nosec
+        SQL = 'select * from {table_identifier} limit 50'.format(
+            table_identifier=sql_utils.Identifier(table.GlueDatabaseName, table.GlueTableName)
+        )  # nosec
```

Comment:
Thanks for fixing this issue! I see the way that you fixed it. Basically you created a custom check to do format securely! Nice!

Just one very minor thing, can you remove the #nosec comment?